src/main/java/Duke.java

```diff
@@ -100,7 +122,7 @@ private static void markTask(Task[] listOfTasks, String[] words) {
 
     private static void unmarkTask(Task[] listOfTasks, String[] words) {
         int number = Integer.parseInt(words[1]);
-        if (number <= 0 || number > 100 || listOfTasks[number] == null || words.length == 1 || !isInt(words[1])) {
+        if (number <= 0 || number >= MAX_TASKS || listOfTasks[number] == null || words.length == 1 || !isInt(words[1])) {
```

Comment:
Coding Quality : Avoid very long if statements or you can enter it to a new line